tests/admin_scripts/tests.py

```diff
@@ -1340,15 +1338,9 @@ def test_no_database(self):
 
     def test_readonly_database(self):
         """
-        Ensure runserver.check_migrations doesn't choke when a database is read-only
-        (with possibly no django_migrations table).
+        Ensure runserver.check_migrations doesn't choke when a database is read-only.
```

Comment:
Is this still simluating a read-only database if the mocking is removed? Chop "Ensure" per https://code.djangoproject.com/ticket/27392.